src/frontend/src/components/FilterPanel/index.tsx

```diff
@@ -148,6 +154,43 @@ const FilterPanel: FC<Props> = ({
     USER_FEATURE_FLAGS.nextclade_download
   );
 
+  useEffect(() => {
+    const uploadDate = activeFilters.find((e) => e.key === "uploadDate");
+    const collectionDate = activeFilters.find(
+      (e) => e.key === "collectionDate"
+    );
+    const qcStatus = activeFilters.find((e) => e.key === "qcMetrics");
+    const lineage = activeFilters.find((e) => e.key === "lineage");
+    const qcStatuses = qcStatus?.params.multiSelected || [];
+    const lineages = lineage?.params.multiSelected || [];
+
+    const anyFiltersActive: boolean[] = [
```

Comment:
Nit: this eventually collapses into a single boolean with `anyFiltersActive.includes(true)` and that's the only intended use -- right? -- so I think doing something like
```
(!!uploadDate || !!collectionDate || !!qcStatus | !!lineage)
```
gets across the intent slightly better.

Reply:
I like to created intermediate variables sometimes to make the logic more clear. I can take it out though if that is not really coming across.